### PR TITLE
Add option to disable substitutions

### PR DESCRIPTION
**Pull request by mridgway**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -162,7 +162,9 @@ Ycb.prototype = {
             }
         }
 
-        this._applySubstitutions(config);
+        if (!options.noSubstitutions) {
```
Comment (redonkulus) [inline index.js]: its always weird to have a negative key but positive value to enable it. Can we make it `applySubstitutions` and make the default `true` to maintain BC?

Comment (mridgway) [inline index.js]: K

**Comment by redonkulus** [comment]
👍

**Comment by drewfish** [comment]
ditto that :+1:

**Comment by mridgway** [comment]
This is actually merged into master and pushed manually, but Github is being slow.

**Pull request closed without merging**